reframe/schemas/runreport.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/eth-cscs/reframe/master/schemas/runreport.json",
+    "title": "Validation schema for ReFrame's run report",
+    "type": "object",
+    "properties": {
+        "reframe_info": {
+            "type": "object",
+            "properties": {
+                "version": {"type": "string"},
+                "command": {"type": "string"},
+                "user": {"type": "string"},
+                "host": {"type": "string"},
+                "check_search_path": {"type": "string"},
```

Comment:
I am not sure we need this, because on the same ground why not store all the configuration parameters. We do need to store the configuration file instead.